services/API-service/src/scripts/seed-point-data.ts

```diff
@@ -72,8 +77,13 @@ export class SeedPointData implements InterfaceScript {
         countryCodeISO3,
         pointDtos,
       );
-    } catch {
-      return Promise.resolve();
+    } catch (error) {
+      // If validation or upload fails, then log and throw error
+      console.error('Error seeding point data:', error);
```

Comment:
Use `this.logger.log` instead of `console.log`. See [SeedAdminAreaData](https://github.com/rodekruis/IBF-system/blob/32a7ac8a02f0794112b4ec588258a525331c5c72/services/API-service/src/scripts/seed-admin-area-data.ts#L21) for reference.